packages/core/src/components/DataTable/helpers/sortData.ts

```diff
@@ -1,20 +1,27 @@
 import { SortDirection, SortDirectionType } from 'react-virtualized';
 import { GenericRow } from '../types';
 
-function sortDesc(a: any, b: any) {
-  if (typeof b === 'undefined' || a < b) {
+// https://stackoverflow.com/questions/29829205/sort-an-array-so-that-null-values-always-come-last
+function sort(a: any, b: any, ascending: boolean = false) {
+  // Equal items sort equally.
+  if (a === b) {
+    return 0;
+  }
+  // null sort after anything else.
+  /* eslint-disable no-eq-null */
+  if (a == null) {
```

Comment:
This does not distinguish `null` and `undefined`.

Does it ensure that `[undefined, null, undefined, null]` will become `[undefined, undefined, null, null]`?

Reply:
yeah - we had a back and forth about it. right now, not distinguishing between `null` and `undefined`. it'll sort `null` before `undefined`. so, `[null, null, undefined, undefined]`